integration/db/db_integration_test.go

```diff
@@ -96,6 +99,96 @@ func TestDatabaseAccessSeparateListeners(t *testing.T) {
 	t.Run("MongoSeparateListener", pack.testMongoSeparateListener)
 }
 
+// testIPPinning tests a scenario where a user with IP pinning
+// connects to a database
+func (p *DatabasePack) testIPPinning(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{
+		TestBuildType: modules.BuildEnterprise,
+		TestFeatures:  modules.Features{DB: true},
+	})
+
+	type testCase struct {
+		desc          string
+		targetCluster databaseClusterPack
+		pinnedIP      string
+		wantClientErr string
+	}
+
+	testFunc := func(t *testing.T, tc testCase) {
```

Comment:
Is there a reason not to inline this in the for loop below? I don't feel strongly about this, but it struck me as unusual.

Reply:
Yeah, weird leftover of test experimenting, at first I planned maybe to reuse it for another test, but then decided not to and forgot to inline. Inlined now 👍